src/evt_functions.jl

```diff
@@ -128,13 +128,24 @@ export get_aux_evt_chsel_propfunc
 """
     get_spms_evt_chdata_propfunc(data::LegendData, sel::AnyValiditySelection)
 
-Get the Ge-detector channel data output PropertyFunction.
+Get the SiPM-detector channel data output PropertyFunction.
 """
 function get_spms_evt_chdata_propfunc(data::LegendData, sel::AnyValiditySelection)
     ljl_propfunc(_dataprod_evt(data, sel, :spms).chdata_output)
 end
 export get_spms_evt_chdata_propfunc
 
+"""
+    get_spms_evt_kwargs(data::LegendData, sel::AnyValiditySelection)
+
+Get the SiPM-detector evt kwargs.
+"""
+function get_spms_evt_kwargs(data::LegendData, sel::AnyValiditySelection)
+    kwargs = _dataprod_evt(data, sel, :spms).kwargs
+    NamedTuple([(k, if v isa String Symbol(v) else v end) for (k, v) in pairs(kwargs)])
```

Comment:
```suggestion
    NamedTuple([(k, Symbol(v)) for (k, v) in pairs(kwargs)])
```
? 
or what could v be where you need something else